src/main/java/net/dries007/tfc/client/render/animal/LayerCamelDecor.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Work under Copyright. Licensed under the EUPL.
+ * See the project README.md and LICENSE.txt for more information.
+ */
+
+package net.dries007.tfc.client.render.animal;
+
+import javax.annotation.ParametersAreNonnullByDefault;
+
+import net.minecraft.client.renderer.entity.layers.LayerRenderer;
+
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+
+import net.dries007.tfc.client.model.animal.ModelCamelTFC;
+import net.dries007.tfc.objects.entity.animal.EntityCamelTFC;
+
+import static net.dries007.tfc.TerraFirmaCraft.MOD_ID;
+
+
+@SideOnly(Side.CLIENT)
+@ParametersAreNonnullByDefault
+public class LayerCamelDecor implements LayerRenderer<EntityCamelTFC>
+{
+    private static final ResourceLocation[]CAMEL_DECOR_TEXTURES = new ResourceLocation[]{new ResourceLocation(MOD_ID, "textures/entity/animal/decor/white.png"), new ResourceLocation(MOD_ID, "textures/entity/animal/decor/orange.png"), new ResourceLocation(MOD_ID, "textures/entity/animal/decor/magenta.png"), new ResourceLocation(MOD_ID, "textures/entity/animal/decor/light_blue.png"), new ResourceLocation(MOD_ID, "textures/entity/animal/decor/yellow.png"), new ResourceLocation(MOD_ID, "textures/entity/animal/decor/lime.png"), new ResourceLocation(MOD_ID, "textures/entity/animal/decor/pink.png"), new ResourceLocation(MOD_ID, "textures/entity/animal/decor/gray.png"), new ResourceLocation(MOD_ID, "textures/entity/animal/decor/silver.png"), new ResourceLocation(MOD_ID, "textures/entity/animal/decor/cyan.png"), new ResourceLocation(MOD_ID, "textures/entity/animal/decor/purple.png"), new ResourceLocation(MOD_ID, "textures/entity/animal/decor/blue.png"), new ResourceLocation(MOD_ID, "textures/entity/animal/decor/brown.png"), new ResourceLocation(MOD_ID, "textures/entity/animal/decor/green.png"), new ResourceLocation(MOD_ID, "textures/entity/animal/decor/red.png"), new ResourceLocation(MOD_ID, "textures/entity/animal/decor/black.png")};
```

Comment:
```suggestion
    private static final ResourceLocation[] CAMEL_DECOR_TEXTURES = IntStream.range(0, 16).map(i -> new ResourceLocation(MOD_ID, "textures/entity/animal/decor/" + EnumDyeColor.byMetadata(i).etName() + ".png").toArray(ResourceLocation[]::new);
```
is much cleaner imo.